### DISCUSSION TITLE
Relation to the same table

### DISCUSSION THREAD

**Discussion by oSethoum** [pocketbase/pocketbase#2009] [Q&A] [answered]
How can I have relation to the same table? example of a use case, a seller can invite other sellers using his invite code.

**Comment by ganigeorgiev** [comment, marked as answer]
You can add a self-referencing `relation` in the same collection, but the caveat is that the collection must be first created/saved so that we can use its id in the relation field options. In other words:

1. Create the collection without the self-referencing `relation` field.
2. Update the collection by adding the self-referencing `relation` field.

_(this eventually will be simplified in the UI and/or done behind the scenes)_

**Reply by oSethoum** [reply]
Thanks you for the quick answer, apparently it was available in the collection select but didn't appear before, thanks a lot.
